x/mock/simulation/random_simulate_blocks.go

```diff
@@ -402,6 +402,7 @@ func RandomRequestBeginBlock(r *rand.Rand, params Params, validators map[string]
 			// offline
 			signed = false
 		}
+		eventFn("beginblock/signing/signed", signed)
```

Comment:
Is this a useful stat? I really don't think it is, since the information it provides is unhelpful after a very small number of blocks.

Reply:
I think it's still worth recording for an idea of the overall distribution.

Reply:
I'm dubious. I'd opt to remove just b/c it also skews the "number of operations" count, though I guess we could create a fix for this. 

This is essentially looking at the result of a binomial distribution after a very large number of runs. Maybe we make this one display the percentage between the two, and not the raw number?